.changeset/swift-squids-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': minor
```

Comment:
This are just patch changes :)

Reply:
I've updated the changeset to be a patch now, sorry for the confusion